src/NodeAnalyzer/PropertyFetchAnalyzer.php

```diff
@@ -96,6 +96,9 @@ public function containsLocalPropertyFetchName(Trait_ $trait, string $propertyNa
         );
     }
 
+    /**
+     * @phpstan-assert-if-true PropertyFetch|StaticPropertyFetch $node
```

Comment:
since we assert at the method definition, we don't need a inline-@var on each call-site